config/default/core.entity_form_display.node.judgement.default.yml

```diff
@@ -0,0 +1,357 @@
+uuid: d6308cb8-4066-40f5-934b-43d0c2622688
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.judgement.body
+    - field.field.node.judgement.field_case_history
+    - field.field.node.judgement.field_casename
+    - field.field.node.judgement.field_casenumber
+    - field.field.node.judgement.field_cases_considered
+    - field.field.node.judgement.field_court_name
+    - field.field.node.judgement.field_external_link
```

Comment:
Can we re-use the existing legislation external link here?

Reply:
I believe this *is* re-using the existing field, since the name `field_external_link` must be unique.

Reply:
Correct.